### ISSUE TITLE
Tutorial document: yolo_voc

### ISSUE THREAD

**Issue by WHCer** [dmlc/gluon-cv#236]
it seems that the yolov3 in model zoo' training script doesn't match the training log and the title.
for example:
yolo3_darknet53_voc @ 320x320, 
training script:
python3 ~/gluon-cv/scripts/detection/yolo/train_yolo3_rand_size.py --dataset coco --gpus 0,1,2,3,4,5,6,7 --batch-size 64 -j 8 --log-interval 100 --lr-decay-epoch 220,250 --epochs 280 --syncbn
use coco datasets.
Furthermore, in the log:
Namespace(batch_size=64, data_shape=416, dataset='voc', epochs=200, gpus='0,1,2,3,4,5,6,7', log_interval=100, lr=0.001, lr_decay=0.1, lr_decay_epoch='160,180', momentum=0.9, network='darknet53', num_samples=16551, num_workers=16, resume='', save_interval=10, save_prefix='yolo3_darknet53_voc', seed=233, start_epoch=0, syncbn=True, val_interval=1, wd=0.0005)
It seems that the test size is 416 ratherthan 320, so the @320x320 doesn't make sense, maybe is single scale training?
Could ablation studies and the result of darknet be provided to show the reliability of the implements? In fact, I see many different setting with the darknet, ex: init, loss, data augmentation.....

**Comment by zhreshold** [comment]
I am still preparing the tutorials, but regarding you question, the model is trained with various shapes from 320 to 608, and evaluated in 416.

After training, I used gluon-cv/scripts/detection/yolo/eval_yolo.py to get results @ 320x320.

**Comment by WHCer** [comment]
Thanks.
